File: Week2/day02/coding-hours.py
average_hours = 6
semester_length = 17
number_of_workdays = 5
work_hours_weekly = 52
all_working_hours = work_hours_weekly * semester_length
print(all_working_hours)
Hours_spent_coding_in_a_semester = print("Hours spent coding in a semester: " + str(average_hours * semester_length * number_of_workdays))


average_work_hours_weekly = 52
print("Percentage of the coding hours in the semester: " + str((average_hours * semester_length * number_of_workdays) / all_working_hours) * 100)

File: Week7/day02/coding-hours.js
'use strict';

// An average Green Fox attendee codes 6 hours daily
// The semester is 17 weeks long
//
// Print how many hours is spent with coding in a semester by an attendee,
// if the attendee only codes on workdays.
//
// Print the percentage of the coding hours in the semester if the average
// work hours weekly is 52

var averageHours = 6;
var semesterLength = 17;
var workdaysInAWeek = 5;
var averageWorkHoursWeekly = 52;

var hoursSpentCoding = (averageHours * semesterLength * workdaysInAWeek);
console.log(hoursSpentCoding);

var percentageOfCodingHours = hoursSpentCoding * 100 / (averageWorkHoursWeekly * semesterLength);
console.log(percentageOfCodingHours);
